### PR TITLE
Don't use ResourceManager

### LINKED ISSUE #362: pretzel create is broken if used as global tool on netcoreapp3.0 runtime
**Describe the bug**
When running create on dotnetcore as a global tool on netcoreapp3.0 creation is broken.

**To Reproduce**
1. Install dotnetcore3.0
2. Install pretzel as a global tool
3. create an empty directory
4. run pretzel create

**Expected behavior**
The recipe is installed correctly

**Actual behavior**
The files are created, but every file contains something like: 
```txt
..\Resources\Razor\Atom.cshtml;System.String, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
```

### PR DESCRIPTION
**Pull request by biohazard999**
This fixes #362

### CODE REVIEW

File: src/Pretzel.Logic/Recipes/Recipe.cs [hunk 1]
```diff
@@ -77,18 +75,17 @@ public void Create()
...
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"_config.yml"), Properties.Liquid.Config);
-                    fileSystem.File.WriteAllText(Path.Combine(directory, @"_includes", "head.html"), Properties.Liquid.Head);
+                    CreateFile(@"Resources\Liquid\Rss.liquid", directory, @"rss.xml");
+                    CreateFile(@"Resources\Liquid\Atom.liquid", directory, @"atom.xml");
+                    CreateFile(@"Resources\Liquid\Sitemap.liquid", directory, @"sitemap.xml");
+                    CreateFile(@"Resources\Liquid\Layout.liquid", directory, @"_layouts", "layout.html");
+                    CreateFile(@"Resources\Liquid\Post.liquid", directory, @"_layouts", "post.html");
+                    CreateFile(@"Resources\Liquid\Index.liquid", directory, @"index.html");
+                    CreateFile(@"Resources\Liquid\About.liquid", directory, @"about.md");
+                    fileSystem.File.WriteAllText(Path.Combine(directory, @"_posts", string.Format("{0}-myfirstpost.md", DateTime.Today.ToString("yyyy-MM-dd"))), GetResourceString(@"Resources\Liquid\FirstPost.liquid"));
```
Comment (laedit) [inline src/Pretzel.Logic/Recipes/Recipe.cs]: This one can't use `CreateFile`?

**Review by laedit — DISMISSED** [review]
Thanks a lot for this PR and welcome back, it is always nice to see you here! :)

File: src/Pretzel.Logic/Recipes/Recipe.cs (hunk 1 above)
Comment (biohazard999) [reply src/Pretzel.Logic/Recipes/Recipe.cs]: Totally overlooked that one, was irritated by the `string.format`

Comment (biohazard999) [reply src/Pretzel.Logic/Recipes/Recipe.cs]: Should be fixed now.

**Pull request closed without merging**